api/proto/config/storage.proto

```diff
@@ -193,7 +200,8 @@ message Volume {
   int64 maxsizebytes = 5;
   bool readonly = 6;       // Will be offered to tasks as read-only
   string displayName = 7;  // Optional friendly name echo'ed in info message
-  bool clear_text = 8;  // Flag to indicate the volume encryption needed or not
+  bool clear_text = 8;     // Flag to indicate the volume encryption needed or not
+  Target target = 9;       // How the volume will be presented to the instance. At the moment only the AppCustom is used
```

Comment:
Should it be an option inside `VolumeRef` message, as a way we are referencing to the Volume to access?

Reply:
I think it is more logical here, as it describes the qualities of the volume itself. I mean if it is a tarball, there are not many reasons to attach it as virtual disk. But I don't feel very strong about that